autogen.sh

```diff
@@ -6,27 +6,42 @@
 srcdir=`dirname $0`
 (test -d $srcdir/m4) || mkdir $srcdir/m4
 
-pushd $srcdir > /dev/null
-gtkdocize && \
-autoreconf --verbose --force --install --make || {
- echo 'autogen.sh failed';
- exit 1;
-}
-
-popd > /dev/null
-
-while test "x$@" != "x" ; do
-optarg=`expr "x$@" : 'x[^=]*=\(.*\)'`
```

Comment:
# I don't see any uses of `optarg` variable here. Am I wrong? AFAIK `getopts` uses `OPTARG` but there is no `getopts` here

# Both lines 18 and 19 issued bash errors with typical argument sets, so I decided to get rid of them to prevent real error from passing thorugh